src/Lecture1.hs

```diff
@@ -108,7 +123,15 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+strSum :: [Char] -> Int
+strSum str = accum 0 (words str)
+    where
+        accum :: Int -> [String] -> Int
+        accum agg list =
+            if null list 
+            then agg 
+            else accum (agg + read (head list)) (tail list)
+            
```

Comment:
There's already a standard function `sum` to find the sum of all elements in the list. So you don't need to write your own function 🙂

You can use `map` to apply `read` to every element of the list, and call `sum` on the result. This way, you'll be able to implement `strSum` as a one-liner 🙂